tests/test_counts_templates.py

```diff
@@ -147,4 +169,6 @@
 )
 def test_count_query(expected, kwargs):
     query = get_count_query("test_table", "test_source", ["age", "sex"], **kwargs)
+    with open("output.sql", "w") as f:
+        f.write(query)
```

Comment:
:scissors:?

Reply:
'how to identify which thing i was debugging right before i opened a pr' for 100, alex